src/EditorFeatures/Core/Implementation/CallHierarchy/CallHierarchyDetail.cs

```diff
@@ -102,7 +102,8 @@ public void NavigateTo()
             if (document != null)
             {
                 var navigator = _workspace.Services.GetService<IDocumentNavigationService>();
-                navigator.TryNavigateToSpan(_workspace, document.Id, _span, usePreviewTab: true);
+                var options = _workspace.Options.WithChangedOption(NavigationOptions.UsePreviewTab, true);
```

Comment:
What if the user does want this option off? Is this just working around that you didn't set a default of true for your existing option?